infrastructure/analytics/analytics.go

```diff
@@ -17,32 +17,62 @@
 package analytics
 
 import (
+	"encoding/json"
 	"sync"
+	"time"
 
+	"github.com/google/uuid"
+	"github.com/snyk/go-application-framework/pkg/analytics"
 	configuration2 "github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/instrumentation"
 	localworkflows "github.com/snyk/go-application-framework/pkg/local_workflows"
 	"github.com/snyk/go-application-framework/pkg/workflow"
 
 	"github.com/snyk/snyk-ls/application/config"
+	"github.com/snyk/snyk-ls/internal/types"
+	"github.com/snyk/snyk-ls/internal/util"
 )
 
 var analyticsMu = sync.RWMutex{}
 
 func SendAnalyticsToAPI(c *config.Config, payload []byte) error {
 	logger := c.Logger().With().Str("method", "analytics.sendAnalyticsToAPI").Logger()
-	logger.Debug().Str("payload", string(payload)).Msg("Analytics Payload")
 
-	inputData := workflow.NewData(
-		workflow.NewTypeIdentifier(localworkflows.WORKFLOWID_REPORT_ANALYTICS, "reportAnalytics"),
-		"application/json",
-		payload,
-	)
+	var eventsParam types.AnalyticsEventParam
```

Comment:
It tries to get a struct - unmarshalling into a struct does not work, we use the old flow, passing the data through to analytics service.